.gitignore

```diff
@@ -11,7 +11,9 @@ coverage.xml
 .ipynb_checkpoints
 .mypy_cache
 .vscode
+.DS_Store
 [.]venv/
 __pycache__/
 doc/scapy/_build
 doc/scapy/api
+scapy/.DS_Store
```

Comment:
I think this can be dropped. `.DS_Store` (without slashes) should already match at any level (including `scapy/`).

Reply:
Done.

Thank you.